src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { QUESTIONS } from "../../constants/constants";
+
+const HomePage = () => {
+  const navigate = useNavigate();
+
+  const [questionsAmount, setQuestionsAmount] = useState(1);
+  const [error, setError] = useState(false);
+
+  const fullQuestionsLength = QUESTIONS.length;
+
+  const handleSubmit = (e: React.MouseEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (questionsAmount !== 0 && questionsAmount <= fullQuestionsLength) {
```

Comment:
Seems that I can enter "-10" questions and will be able to submit...

Reply:
fixed